KingPhisher

```diff
@@ -33,75 +33,75 @@
 import argparse
 import logging
 import os
+import subprocess
 import sys
-import threading
-import time
 
-if getattr(sys, 'frozen', False):
-	# set the basemap data directory for frozen builds
-	os.environ['BASEMAPDATA'] = os.path.join(os.path.dirname(sys.executable), 'mpl-basemap-data')
-
-from king_phisher import color
-from king_phisher import find
-from king_phisher import utilities
-from king_phisher import version
-from king_phisher.client import application
-from king_phisher.client import gui_utilities
-
-from gi.repository import GObject
-from gi.repository import Gtk
+from king_phisher import startup
 
 def main():
 	parser = argparse.ArgumentParser(description='King Phisher Client GUI', conflict_handler='resolve')
-	utilities.argp_add_args(parser, default_root='KingPhisher')
-	parser.add_argument('-c', '--config', dest='config_file', required=False, help='specify a configuration file to use')
-	parser.add_argument('--no-plugins', dest='use_plugins', default=True, action='store_false', help='disable all plugins')
-	parser.add_argument('--no-style', dest='use_style', default=True, action='store_false', help='disable interface styling')
-	arguments = parser.parse_args()
+	startup.argp_add_wrapper(parser)
+	startup.argp_add_client(parser)
+	startup.argb_add_default_args(parser)
```

Comment:
It looks like `argb_add_default_args` should be `argp_add_default_args`. Not sure why that one is argB.